QuadPaySDK/Configuration/QuadPaySDKProduction.plist

```diff
@@ -3,10 +3,10 @@
 <plist version="1.0">
 <dict>
 	<key>QUADPAY_JS_URL</key>
-	<string>https://cdn.us.zip.co/v1/quadpay.js</string>
+	<string>https://cdn.dev.us.zip.co/v1/quadpay.js</string>
 	<key>CDN_URL</key>
 	<string>https://qpmerchconfigsprd.blob.core.windows.net/</string>
 	<key>GATEWAY_URL</key>
-	<string>https://gateway.us.zip.co/</string>
+	<string>https://gateway.dev.us.zip.co/</string>
```

Comment:
Why have these changed?

Reply:
i changed those because i had to do a release from branch for dev , so i will revert that